bindings/protocols/coap/index.html

```diff
@@ -54,86 +54,280 @@ <h2>Introduction</h2>
+                of <em>features</em> makes more sense than offering a generic format for defining them
+                within a TD form, also considering that there are options with binary, numeric or
+                string-based values (see sections 3.1 and 3.2 of [[RFC7252]] for more details).
+            </p>
+        </aside>
+        <section>
+            <h3>Requests/Responses</h3>
+            <p>
+                Things accessible via CoAP provide interaction affordances that direct Consumers to send CoAP requests to a CoAP endpoint. The Thing processes these requests and sends appropriate CoAP responses.
+            </p>
+            <p>
+                The <strong>target resource</strong> is specified in the Thing Description by the <code>href</code> member of a form, using one of the CoAP URI schemes (see section [[[#url]]]).
+                The <strong>request method</strong> (e.g., <code>"GET"</code>, <code>"PUT"</code>, <code>"POST"</code>, or <code>"DELETE"</code>) is specified using the <code>cov:method</code> member of a form.
+            </p>
+            <pre id="example-request" class="example" title="A simple Thing Description using CoAP">
+{
+    "@context": "https://www.w3.org/2022/wot/td/v1.1",
+    ...
+    "properties": {
+        "status": {
+            "type": "string",
+            "readOnly": true,
+            "forms": [{
+                "cov:method": "GET",
+                "href": "coap://[2001:DB8::1]/status",
+                "contentType": "text/plain;charset=utf-8",
+                "op": ["readproperty"]
+            }]
+        }
+    }
+}
+</pre>
+            <p>
+                The <code>cov:method</code> member may be omitted if the request method is the default for the operation (see section [[[#default-mappings]]]).
+            </p>
+            <pre id="example-request-defaults" class="example" title="A simple Thing Description using CoAP with defaults omitted">
+{
+    "@context": "https://www.w3.org/2022/wot/td/v1.1",
+    ...
+    "properties": {
+        "status": {
+            "type": "string",
+            "readOnly": true,
+            "forms": [{
+                "href": "coap://[2001:DB8::1]/status",
+                "contentType": "text/plain;charset=utf-8"
+            }]
+        }
+    }
+}
+</pre>
+            <p>
+                The <code>contentType</code> and <code>contentCoding</code> members of a form specify the CoAP content format of the request and/or response.
+                Consumers are expected to be able to map the combinations of content type and content coding they understand to the corresponding CoAP content format IDs listed in the <a href="https://www.iana.org/assignments/core-parameters/#content-formats">IANA CoAP Content-Formats registry</a>.
+                TD Producers are adviced to only use combinations of content types and content codings in CoAP forms that are mappable to registered content format IDs.
```

Comment:
Oh, I wasn't sure if you could use assertions in binding documents. I think we could also consider making this a MUST, as otherwise consumers might need to fall back to a default value.

Should the sentence before also become an assertion?

Reply:
I don't think we should put normative text in the examples section 🙂

I'm also not sure if we need to put any normative statements on this issue in the document. In the end, a consumer can only make use of an affordance if (1) it understands the content type/content coding to be used and (2) that combination has a content format ID. There is no harm if the producer puts a combination that the consumer doesn't understand or that doesn't have an ID (yet).

@JKRhb What do you mean by "consumers might need to fall back to a default value"?